aws_lambda_powertools/utilities/parameters/secrets.py

```diff
@@ -116,6 +119,79 @@ def _get_multiple(self, path: str, **sdk_options) -> Dict[str, str]:
         """
         raise NotImplementedError()
 
+    def _set(
+        self,
+        name: str,
+        value: Union[str, dict, bytes],
+        *, # force keyword arguments
+        idempotency_id: Optional[str] = None,
```

Comment:
What is the idea of `idempotency_id`? Is this field an alias for [ClientRequestToken](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/secretsmanager/client/create_secret.html)? If so, I think it might confuse customers because it's not explicitly what idempotency means here. I think we can think about other name here.

Reply:
Changed this to `client_request_token`  so it lines up with the boto argument name.